packages/terra-icon/src/Icon.module.scss

```diff
@@ -21,3 +21,66 @@
     transform: scaleX(-1);
   }
 }
+
+:global {
```

Comment:
Could you make this :local?

Reply:
If we change it to :local, then CSS Modules changes the keyframe name [here](https://github.com/cerner/terra-core/pull/3187/files/138e706ceb6f1bb20470bacc1fb2f13f8937cd3f#diff-3ca2b30d1d2a7907939e40bd50431ce8R78) by appending the fileName and hash key but doesn't do anything to the keyframe declaration. Hence, the keyframe is not recognized.

That's why I have passed it in global scope.

Reply:
Could you make line 75 and below local?

Reply:
Similar behavior is being seen if I put the CSS under the line 75 into local.